sentences/ro/light_HassLightSet.yaml

```diff
@@ -32,6 +34,7 @@ intents:
           - "[<seteaza>] [<culoarea>] [<din>] <name> [<in> | la] [<culoarea>] {color}"
           - "[<seteaza>] <culoarea> [<in> | la] {color} [<din>] <name>"
           - "[<seteaza>] [<in> | la] {color} [<culoarea>] [<din>] <name>"
+          - "[<seteaza>] <name> [<in> | la] [<culoarea>] {color}"
```

Comment:
As opposed to brightness, I think it's fine to be able to say "plafoniera hol verde". So this is OK.